### ISSUE TITLE
gql import package

### ISSUE THREAD

**Issue by radhey-vigilant** [SharmaPawan11/vendure-razorpay-payment-plugin#2]
The line to import `gql` in `razorpay-plugin.ts` is 
`import { gql } from "apollo-server-core";`

So after installing the plugin, an error is being thrown saying "apollo-server-core" module not found. To address that I am having to install the package separately. Do you think importing from `graphql-tag` instead would remove this dependency and need for additional installation? And other vendure related dependencies can be installed as dev dependencies as show here in the example plugin https://github.com/vendure-ecommerce/plugin-template/blob/master/package.json ?

**Comment by SharmaPawan11** [comment]
Do the issue resolve after installing it separately ?

**Comment by radhey-vigilant** [comment]
yeah. Although I am working with an npm workspaces monorepo. Perhaps the issue may be due to that? I am not sure. But installing apollo-server-core separately did resolve it.

**Comment by SharmaPawan11** [comment]
Actually, I developed this plugin before the existence of `plugin-template` repo/guidelines . It is completely possible that now there is a newer and better way to create/import dependencies for vendure plugin. You may raise a pull request after testing your new approach and I'll merge it after testing.

**Comment by radhey-vigilant** [comment]
Alright, I'll do that. thanks.

**Comment by NTAravind** [comment]
hey is razorpay integration good ?

**Comment by SharmaPawan11** [comment]
@radhey-vigilant Can you check on the issue again, I have made some updates to support latest vendure version
